src/bond/james.cljc

```diff
@@ -33,6 +33,22 @@
                       (vec))
      (do ~@body)))
 
+(defmacro with-spy-namespaces
```

Comment:
Good call. I'll go for `with-spy-ns` and `with-stub-ns`. Will merge after those changes.